microcosm_pubsub/batch.py

```diff
@@ -0,0 +1,31 @@
+from marshmallow import fields, Schema
```

Comment:
I had to move these to a separate module or I created a cyclic dependency between the `conventions` and the registry because we do something somewhat weird for the URIPubsubMessage. We don't wire it into the registry and instead we hardcode it as the default in the code itself.